### PR TITLE
Add BrowserStack config file and read in via Gradle, set as system …

### PR DESCRIPTION
**Pull request by mikeSmith08**
…properties and pass to Cucumber tests.

First pass at externalising the config so that its loaded into Gradle and passed to the Cucumber tests. Next step should be to iterate over these BrowserStack capabilities and run a test for each set of values.

### CODE REVIEW

File: src/test/java/uk/gov/dhsc/htbhf/BrowserStackCapabilities.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ * Retrieves and builds the BrowserStack capabilities from the system properties provided by Gradle
+ */
+@Slf4j
+public class BrowserStackCapabilities {
+
+    public static final String BROWSER_STACK_PREFIX = "BROWSERSTACK.TEST.";
+
+    /**
+     * Get the browser stack properties from the System Properties provided, which should have been populated
+     * by the Grale build file all prefixed with BROWSERSTACK.TEST
```
Comment (gethyn1) [inline src/test/java/uk/gov/dhsc/htbhf/BrowserStackCapabilities.java]: `Grale` -> `Gradle`

File: build.gradle [hunk 2]
```diff
@@ -78,3 +78,35 @@ test {
     }
 }
 
+private def setBrowserStackSystemProperties(requiredOsKey, requiredDeviceName) {
+    loadProperties()
+    println "Looking for BrowserStack config: [$requiredOsKey - $requiredDeviceName]"
+    config.eachWithIndex { def osTests, int i ->
```
Comment (YetAnotherMatt) [inline build.gradle]: I don't think you need to use `eachWithIndex` here, as you're not using the index (that I can see). You can just use `each`: http://grails.asia/groovy-each-examples

Comment (YetAnotherMatt) [reply build.gradle]:
```
config.each { osTests ->
```

Comment (mikeSmith08) [reply build.gradle]: Good point, have changed.

File: src/test/java/uk/gov/dhsc/htbhf/BrowserStackCapabilities.java (hunk 1 above)
Comment (mikeSmith08) [reply src/test/java/uk/gov/dhsc/htbhf/BrowserStackCapabilities.java]: Corrected.

**Pull request merged**